entries/n/ngpvan.com.json

```diff
@@ -0,0 +1,16 @@
+{
+  "ActionID": {
+    "domain": "ngpvan.com",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "url": "https://www.ngpvan.com/actionid/",
```

Comment:
Could you move this under the `domain` element?

Reply:
Done. Seems like y'all are particular about the JSON formatting (which is understandable); you might consider putting in an automated check for that to save the back-and-forth 🙂